libraries/engage/favorites/index.js

```diff
@@ -2,7 +2,7 @@
 
 // ACTIONS
 export { default as fetchFavorites } from '@shopgate/pwa-common-commerce/favorites/actions/fetchFavorites';
-export { requestSync as toggleFavorites } from '@shopgate/pwa-common-commerce/favorites/actions/toggleFavorites';
+export { requestForceFavoritesBufferClear as toggleFavorites } from '@shopgate/pwa-common-commerce/favorites/actions/toggleFavorites';
```

Comment:
In file `pwa/libraries/commerce/favorites/index.js` there is another export of `{ requestSync as toggleFavorites }`

Reply:
I changed the name of requestForceFavoritesBufferClear function to requestSync. requestSync was exported in the toggleFavorites file which means it is possible that it is being used by some extensions. There must be a requestSync function to keep the api backward compatible.